core/src/main/java/ai/timefold/solver/core/api/score/analysis/ScoreAnalysis.java

```diff
@@ -53,6 +55,8 @@
 public record ScoreAnalysis<Score_ extends Score<Score_>>(Score_ score,
```

Comment:
Arguably, `ScoreAnalysis` needs an informative `toString()` too.
Something like `Score analysis of score X with Y constraints`.